.github/workflows/snyk.yml

```diff
@@ -1,39 +1,18 @@
-# This action runs Snyk Monitor every day at 6 AM and on every push to main
+# This action runs snyk monitor on every push to main
 name: Snyk
 
 on:
-  schedule:
-    - cron: '0 6 * * *'
   push:
     branches:
-      - master
+      - main
```

Comment:
We will need to migrate this repo over before we can merge this, or we can merge and do that afterwards. What do you think @NovemberTang ?

Reply:
I'm helping @NovemberTang with the Snyk migrations. I think we should hold off until the default branch has been updated. It looks like this project will need to updated in TeamCity to do so.

Reply:
I've changed my mind ... no need to couple these things.